--- sql/gnatcoll-sql_impl.adb.orig	2021-05-20 01:25:55.000000000 -0700
+++ sql/gnatcoll-sql_impl.adb	2021-06-21 09:44:09.437292100 -0700
@@ -188,15 +188,9 @@
         (Self         : Field;
          To           : in out SQL_Field_List'Class;
          Is_Aggregate : in out Boolean)
-      is
-         FC : access SQL_Field_Internal'Class;
-      begin
+      is begin
          if not Self.Data.Is_Null then
-            --  !!! Could not use Element call result in the
-            --  Append_If_Not_Aggregate parameter because of GNAT bug OB03-009
-
-            FC := Self.Data.Get.Element;
-            Append_If_Not_Aggregate (FC, To, Is_Aggregate);
+            Append_If_Not_Aggregate (Self.Data.Get.Element, To, Is_Aggregate);
          end if;
       end Append_If_Not_Aggregate;
    end Data_Fields;
